multimodal/tests/unittests/test_hpo.py

```diff
@@ -10,12 +10,12 @@
 from utils import get_home_dir
 from test_predictor import verify_predictor_save_load
 
-pytest.skip(
-    "HPO testing fails due to RuntimeError: CUDA error: out of memory, "
-    "but the GPU memory usage is low. Something seems wrong with ray. "
-    "Need to investigate the reasons. Skip the tests for now.",
-    allow_module_level=True
-)
+# pytest.skip(
+#     "HPO testing fails due to RuntimeError: CUDA error: out of memory, "
+#     "but the GPU memory usage is low. Something seems wrong with ray. "
+#     "Need to investigate the reasons. Skip the tests for now.",
+#     allow_module_level=True
+# )
```

Comment:
Can we do this in a separate PR after this is merged or do we want to do it now and revert if it is too disruptive? Should we ignore old CI for purposes of merging PRs after this PR is merged?

Reply:
I guess we can do this later, otherwise our current CI will constantly error